src/lit-html.ts

```diff
@@ -167,25 +167,18 @@ export class Template {
     // Edge needs all 4 parameters present; IE11 needs 3rd parameter to be null
     const walker = document.createTreeWalker(
         content,
-        133 /* NodeFilter.SHOW_ELEMENT | NodeFilter.SHOW_COMMENT |
-               NodeFilter.SHOW_TEXT */
+        1157 /* NodeFilter.SHOW_ELEMENT | NodeFilter.SHOW_COMMENT |
+               NodeFilter.SHOW_TEXT | NodeFilter.SHOW_DOCUMENT_FRAGMENT */
```

Comment:
This is so we can backtrack if we're going to remove the first child of the template fragment. This is fine because the template is the only fragment we can ever encounter.